grommet-leaflet/src/themes/base.js

```diff
@@ -1,30 +1,41 @@
-const markerKinds = {
-  default: {
-    container: {
-      // any box props
-      align: 'center',
-      justify: 'center',
-      background: 'background-front',
-      border: {
-        color: 'border',
-        size: 'small',
-      },
-      elevation: 'medium',
-      flex: false,
-      round: 'full',
-      height: { min: '25px', max: '25px' },
-      width: { min: '25px', max: '25px' },
-    },
-    // icon: undefined,
-  },
-};
-
 const base = {
   pin: {
-    ...markerKinds,
```

Comment:
There is a nicety to being able to have `pin` and `cluster` reference the same object for the kinds. Wondering if there's any way we can keep that and just override the `cluster.default.container.height/width` to be `undefined`?

Reply:
Good note, fixed in latest commit